scania-digital-design-system/tegel
# feat(table): default expanded row, expand/collapse methods, tdsChange event

## **Describe pull-request**  
Added a prop for default expanded row.
Added two methods, expand/collapse table row.
Added tdsChange event that includes row id and expand/collapsed state.

## **Issue Linking:**  
- **GitHub:** https://github.com/scania-digital-design-system/tegel/issues/575

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Take a look at preview link
2. Check in console if the row state is changed, also check if you have a random row id or not. Set default expanded and see if the row is open. Invoke custom methods for expand/collapse.

## **Checklist before submission**

## **Suggested test steps**

## **Additional context**  
Unit tests will be added in a day or two.
